database/src/pool/postgres.rs

```diff
@@ -1365,6 +1382,66 @@ where
             .await
             .unwrap();
     }
+
+    async fn insert_benchmark_request(&self, benchmark_request: &BenchmarkRequest) {
+        match &benchmark_request.commit_type {
+            BenchmarkRequestType::Try {
+                sha,
+                parent_sha,
+                pr,
+            }
+            | BenchmarkRequestType::Master {
+                sha,
+                parent_sha,
+                pr,
+            } => {
+                self.conn()
+                    .execute(
+                        r#"
+                        INSERT INTO
```

Comment:
To avoid repeating the SQL query (which always introduces another possibility of making a mistake when the query needs to be modified), I'd suggest adding `tag(&self) -> &str`, `pr(&self) -> Option<u32>, parent(&self) -> Option<&str>` and `commit_type(&self) -> &str` method to `BenchmarkRequest`. Then we can easily just have the query once here.

I thought about making it a prepared statement, but it will probably not be executed often enough for it to be worth it.

Also, I think that inserting `u32` into Postgres will fail, and it needs to be converted to `i32` for the query. Which is why I'd appreciate at least some basic tests :laughing: E.g. at least insert try/master/release artifact into the DB.